lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -208,9 +208,14 @@ def test_not_authorized(self):
         self.assertEqual(response.status_code, 404)
 
 
-class CourseListTests(CourseViewTestsMixin, ModuleStoreTestCase):
+class CourseListTests(CourseViewTestsMixin, SharedModuleStoreTestCase):
     view = 'course_structure_api:v0:list'
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
BTW, the problem with `super()` at the mixin level isn't that it can't reach siblings, it's that it will try to call `object.setUpClass()`, which won't exist.